conf/ephys_upload_job_configs.yml

```diff
@@ -8,6 +8,7 @@ endpoints:
   gcp_prefix:
   codeocean_domain: https://codeocean.allenneuraldynamics.org
 jobs: # Select which jobs to run
+  correct_opto: true
```

Comment:
@alejoe91 is the idea that we would just turn this flag off once OpenEphys starts saving correct locations?

Reply:
Yes that's the idea!